fonts/iconoir.css

```diff
@@ -0,0 +1,3694 @@
+/*!
+ * Iconoir
+ * Copyright (c) 2021 Luca Burgio - https://iconoir.com
+ * License - https://github.com/lucaburgio/iconoir/blob/master/LICENSE (Code: MIT License)
+ * CSS-File created by Till Esser @Wiwaltill
+ */
+
+@font-face {
+    font-family: "Iconoir";
+    src: url("https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.eot");
+    src: url("https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.eot?#iefix") format("embedded-opentype"),
+         url("https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.svg") format("svg"),
+         url("https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.ttf") format("truetype"),
+         url("https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.woff") format("woff");
+    font-weight: normal;
+    font-style: normal;
+}
+
+
+span[class^="iconoir-"]:before, span[class*=" iconoir-"]:before{
```

Comment:
Is it necessary to define the HTML element used? Other icon sets often use the `<i>` element...

One could just use:

```
[class^="iconoir-"]:before, [class*=" iconoir-"]:before{
```

It has the benefit of having even lower CSS specificity of [0, 0, 1, 0] for attribute and pseudo selector:
https://css-tricks.com/specifics-on-css-specificity/

Reply:
Using only an attribute for the selector is probably more "expensive" for the browser and I'd personally use a tag (`i` or `span`).

_I personally don't care too much about the other css rules, the class-icon mapping is all I need._

Reply:
I also think that the browser would load longer.

The "span" element does not differ from the "i" element in usage either. Since the developer of Iconoir has used the "span" element until now, I simply adopted it.